tests/Campaign/DeleteTest.php

```diff
@@ -0,0 +1,57 @@
+<?php
+
+namespace Digitonic\PassonaClient\Tests\Campaign;
+
+use Digitonic\PassonaClient\Entities\Campaigns\Delete;
+use Digitonic\PassonaClient\Exceptions\InvalidData;
+use Digitonic\PassonaClient\Tests\BaseTestCase;
+use GuzzleHttp\Handler\MockHandler;
+use GuzzleHttp\HandlerStack;
+use GuzzleHttp\Psr7\Response;
+use GuzzleHttp\Client;
+use Illuminate\Support\Collection;
+
+/**
+ * @property MockHandler mock
+ * @property HandlerStack handler
+ * @property Client client
+ */
+class DeleteTest extends BaseTestCase
+{
+    protected function setUp(): void
+    {
+        parent::setUp();
+        $this->mock = new MockHandler([
+            new Response(200, [], '[]')
+        ]);
+
+        $this->handler = HandlerStack::create($this->mock);
+
+        $this->client = new Client(['handler' => $this->handler]);
+    }
+
+    /** @test */
+    public function it_can_delete_a_campaign_in_passona()
+    {
+        $passonaApi = new \Digitonic\PassonaClient\Client($this->client);
+
+        $usage = new Delete($passonaApi);
+
+        $response = $usage->delete('5662af3c-f70f-11e9-24ad-0a5864600e04');
```

Comment:
Should this be hardcoded?

Reply:
Its is a fake uuid and is there to serve the purpose of documentation via tests to keep it consistent with real world application / scenarios.